services/gateway/endpoints/src/main/java/org/eclipse/ditto/services/gateway/endpoints/actors/AbstractHttpRequestActor.java

```diff
@@ -65,18 +64,13 @@
 import akka.http.javadsl.model.headers.RawHeader;
 import akka.http.scaladsl.model.ContentType$;
 import akka.http.scaladsl.model.EntityStreamSizeException;
-import akka.japi.Creator;
 import akka.japi.pf.ReceiveBuilder;
 import akka.pattern.AskTimeoutException;
 import akka.util.ByteString;
 import scala.concurrent.duration.Duration;
 import scala.util.Either;
 
-/**
- * Every HTTP Request causes one new Actor instance of this one to be created.
- * It holds the original sender of an issued {@link Command} and tells this one the completed HttpResponse.
- */
-public final class HttpRequestActor extends AbstractActor {
+public abstract class AbstractHttpRequestActor extends AbstractActor {
```

Comment:
Could you please add a class javadoc here?

Reply:
Good point.